pkg/reconciler/serverlessservice/resources/services.go

```diff
@@ -33,16 +33,39 @@ const (
 	servicePortNameH2C   = "http2"
 
 	// ServicePort is the external port of the service
-	servicePort = int32(80)
+	servicePort              = int32(80)
+	serviceHTTP2Port         = int32(81)
```

Comment:
Can we base these on `activator.ServicePort("http1")` and `activator.ServicePort("http2")`?

Reply:
Or just use the constants from there? https://github.com/knative/serving/blob/149aa497a2de0b8e641a54fe93a7e9dcc5fbd9ad/pkg/activator/activator.go#L39-L42

Reply:
I don't want to depend on activator.
I am thinking of moving all this to networking?

Reply:
I'd do this separately, if you don't mind, since this PR does not involve activator code in any way, I'd rather keep it this way.